frontend/model/contracts/group.js

```diff
@@ -29,24 +29,26 @@ export function generateInvites (numInvites: number) {
   }
 }
 
+const groupSettingsValidation = objectOf({
+  // TODO: add 'groupPubkey'
+  groupName: string,
+  groupPicture: string,
+  sharedValues: string,
+  mincomeAmount: number,
+  incomeCurrency: string,
+  proposals: objectOf({
+    [PROPOSAL_INVITE_MEMBER]: proposalSettingsType,
+    [PROPOSAL_REMOVE_MEMBER]: proposalSettingsType,
+    [PROPOSAL_GROUP_SETTING_CHANGE]: proposalSettingsType,
+    [PROPOSAL_PROPOSAL_SETTING_CHANGE]: proposalSettingsType,
+    [PROPOSAL_GENERIC]: proposalSettingsType
+  })
+})
+
```

Comment:
Why were these moved out? These should be left where they were unless there's a very good reason to move them

Reply:
I was trying to reuse validators, but then realized it wasn't needed, so I forgot to rollback it. done.